maintainers/maintainer-list.nix

```diff
@@ -1360,6 +1360,11 @@
       fingerprint = "BF8B F725 DA30 E53E 7F11  4ED8 AAA5 0652 F047 9205";
     }];
   };
+  appsforartists = {
+    github = "appsforartists";
+    githubId = 926648;
+    name = "Brenton Simpson";
+  };
```

Comment:
please also add some way of contacting you, so email or matrix.

Reply:
I check GitHub more reliably than I check my email. 🙃 

The comment at the top of `maintainers-list.nix` says `githubId` is an acceptable form of communication.  Is that out of date?

Reply:
oh wait maybe I misremembered :see_no_evil: